src/libnnstreamer-edge/nnstreamer-edge-common.c

```diff
@@ -15,9 +15,19 @@
 
 #include "nnstreamer-edge-common.h"
 
+/**
+ * @brief Free allocated memory.
+ */
+void
+nns_edge_free (void *data)
```

Comment:
There is a macro, ```SAFE_FREE```. Do you have any reason to define a new function?

Reply:
Related PR is https://github.com/nnstreamer/nnstreamer-edge/pull/32, this is for the release callback.